src/components/VersionSelect/index.tsx

```diff
@@ -61,14 +63,14 @@ class VersionSelectBase extends React.Component<PublicProps> {
 
     return (
       <React.Fragment>
-        {withLeftArrow && (
-          <div className={styles.arrow}>
-            <FontAwesomeIcon icon="long-arrow-alt-left" />
-          </div>
-        )}
-
-        <Form.Group as={Col} className={className}>
-          <Form.Label>{label}</Form.Label>
+        <Form.Group
+          as={Col}
+          className={makeClassName(className, styles.formGroup)}
+        >
```

Comment:
You could use the `controlId` prop here, instead of passing it to each component below: https://react-bootstrap.github.io/components/forms/#form-group-props

Reply:
Ooh, fancy. Thanks for the tip.